jujugui/static/gui/src/app/components/user-profile/entity/entity.js

```diff
@@ -241,19 +241,42 @@ YUI.add('user-profile-entity', function() {
     },
 
     /**
-      Generate the owner for a model.
+      Generate the credentials for a model.
 
-      @method _generateOwner
-      @return {Object} The owner markup.
+      @method _generateCredentials
+      @return {Object} The credential markup.
     */
-    _generateOwner: function() {
-      const entity = this.props.entity;
-      if (this.props.type !== 'model' || !entity.owner) {
-        return;
+    _generateModelInfo: function() {
+      if (this.props.type !== 'model') {
+        return null;
       }
+      const model = this.props.entity;
+      // See the _generateRow function in model-list.js for matching logic.
+      // Both sections should be kept roughly in sync.
+      const region = model.region || 'no region';
+      let owner = '--';
+      if (model.owner) {
+        owner = model.ownder = model.owner.split('@')[0];
```

Comment:
I don't like how a UI component is modifying the value of the owner for the rest of the app. Was it intentional to remove the domain for the owner for everywhere in the app?